cmd/podman/create.go

```
@@ -195,11 +195,12 @@ func createCmd(c *cli.Context) error {
 	logrus.Debug("new container created ", ctr.ID())
 
 	if c.String("cidfile") != "" {
-		libpod.WriteFile(ctr.ID(), c.String("cidfile"))
-	} else {
-		fmt.Printf("%s\n", ctr.ID())
+		err := libpod.WriteFile(ctr.ID(), c.String("cidfile"))
+		if err != nil {
```

Nit do this on one line.

if err := ...

---

Why this not a failure?  I think you should return the error.